fastembed/text/e5_onnx_embedding.py

```diff
@@ -25,6 +25,16 @@
             "hf": "xenova/paraphrase-multilingual-mpnet-base-v2",
         },
     },
+    {
+        "model": "intfloat/multilingual-e5-large-instruct",
+        "dim": 1024,
+        "description": "multilingual model, e5-large-instruct",
+        "size_in_GB": 1.03,
+        "sources": {
+            "url": "https://drive.google.com/file/d/1XSl4RantOvSTiZeta7tl8S6f9P6MMvSo/view?usp=sharing",
```

Comment:
This is not a valid download URL. You can isolate and reproduce this by trying to download the model using requests.